scrapers/scrape_vs.py

```diff
@@ -31,5 +29,30 @@
     dd.deaths = row['Cumul décès COVID-19']
     if row['Nb de nouvelles sorties'] is not None:
         dd.recovered = sum(r['Nb de nouvelles sorties'] for r in rows[:i+1])
-    print(dd)
+    data.append(dd)
+
+# parse weekly data for isolated and quarantined numbers
+base_url = 'https://www.vs.ch'
+stat_url = base_url + '/de/web/coronavirus/statistiques'
+content = sc.download(stat_url, silent=True)
+soup = BeautifulSoup(content, 'html.parser')
+res = soup.find(string=re.compile(r'Synthese COVID19 VS Woche\d+')).find_previous('a')
+weekly_pdf_url = base_url + res.attrs['href']
+content = sc.pdfdownload(weekly_pdf_url.replace(' ', '%20'), silent=True)
 
+# add isolated/quarantined to the existing DayData item
+week_end_date = sc.find(r'vom (\d+)\. bis (\d+\.\d+\.20\d{2})', content, group=2)
+week_end_date = sc.date_from_text(week_end_date).isoformat()
+for dd in data:
+    if dd.datetime == week_end_date:
```

Comment:
Why do you loop over the `data` list? I would simply create new DayData objects with the week_end_date as it's `datetime`. There is no need to reuse the existing objects. Should for whatever reason a date be missing in the xls, this would fail to add the isolated and quarantined numbers.

Reply:
I implemented it like that in #1024, but there was a problem with the CI check... I'm not really sure, what's wrong with the changes there (or the check is "wrong").

Reply:
Ah yes, the check is a bit stupid. It only runs on the last entry (as the assumption is, that the last entry is the most complete). This is mostly to be able to add an expected value (e.g. `deaths`) but not fail, if at any point in the past, this value was not provided.
I would simply change the order, so first scrape the weekly pdf, and then scrape the xls. That way, there shouldn't be a problem.

Reply:
Ah that was it... I've created a new PR #1027

Reply:
why a new PR? you could just update this one. Then the conversation is not lost.

Reply:
To have a cleaner history. I've reopened it and moved the isolated/quarantined parsing to the top of the file.

Reply:
If you want, you can always rebase your branch, and remove squash etc. your commits. It's up to you. If you prefer to create a new PR, I don't mind. 👍